server/models/Booking.js

```diff
@@ -14,4 +16,6 @@ const bookingSchema = new Schema(
   { timestamps: true }
 );
 
+const Booking = mongoose.model('Booking', bookingSchema);
```

Comment:
then this would be changed to:

```suggestion
const Booking = model('Booking', bookingSchema);
```